From: Hannes Reinecke <hare@suse.de>
Date: Wed, 12 Jun 2013 16:21:07 +0200
Subject: scsi: kABI fixes
Patch-Mainline: n/a
References: bnc#798050

'manufacturer' is unused in the SLES11 SP2 code, so we can
re-use it for 'eh_timeout'.

Signed-off-by: Hannes Reinecke <hare@suse.de>
---
 include/scsi/scsi_device.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/include/scsi/scsi_device.h b/include/scsi/scsi_device.h
index a4fa78a..f4618c2 100644
--- a/include/scsi/scsi_device.h
+++ b/include/scsi/scsi_device.h
@@ -92,8 +92,12 @@ struct scsi_device {
 
 	unsigned int id, lun, channel;
 
+#ifndef __GENKSYMS__
+	unsigned int eh_timeout; /* Error handling timeout */
+#else
 	unsigned int manufacturer;	/* Manufacturer of device, for using 
 					 * vendor-specific cmd's */
+#endif
 	unsigned sector_size;	/* size in bytes */
 
 	void *hostdata;		/* available to low-level driver */
@@ -113,7 +117,6 @@ struct scsi_device {
 				 * scsi_devinfo.[hc]. For now used only to
 				 * pass settings from slave_alloc to scsi
 				 * core. */
-	unsigned int eh_timeout; /* Error handling timeout */
 	unsigned writeable:1;
 	unsigned removable:1;
 	unsigned changed:1;	/* Data invalid due to media change */
-- 
1.7.12.4

